lib/table_for/base.rb

```diff
@@ -132,6 +132,8 @@ def header_sort_link(column, options={}, &block)
     def translation_lookup_prefix
       if global_options[:records].respond_to?(:model)
         "activerecord.attributes.#{global_options[:records].model.to_s.underscore}"
+      elsif global_options[:records].all? {|record| record.is_a?(ApplicationDecorator) && record.class == global_options[:records].first.class && record.respond_to?(:object) }
```

Comment:
This will break if the application does not have ApplicationDecorator defined. Perhaps the elsif should start with something like:

```
elsif defined?(ApplicationDecorator) && global_options[:records].all? {|record| record.is_a?(ApplicationDecorator) && record.class == global_options[:records].first.class && record.respond_to?(:object) }
 

 
This will break if the application does not have ApplicationDecorator defined. Perhaps the elsif should start with something like:

```
elsif defined?(ApplicationDecorator) && 

Attach files by dragging & dropping, selecting or pasting them.
        "activerecord.attributes.#{global_options[:records].first.object.class.to_s.underscore}"
      elsif global_options[:records].all? {|record| record.is_a?(ActiveRecord::Base) && record.class == global_options[:records].first.class }
        "activerecord.attributes.#{global_options[:records].first.class.to_s.underscore}"
      elsif global_options[:records].all? {|record| record.class == global_options[:records].first.class }
```

This would also be dependent on whether ApplicationDecorator is expected to exist when using the draper gem (which I haven't used in several years so I can't remember).

Also, please add a test case to handle this scenario.